plugins/poll/services/PollService.php

```diff
@@ -98,7 +98,9 @@ public function voteAction($pollId, $userId, $answerIds)
 		KalturaResponseCacher::disableCache();
 		try
 		{
-			return PollActions::setVote($pollId, $userId, $answerIds);
+			$pollActions = new PollActions();
+			$ksUserId = kCurrentContext::getCurrentKsKuserId();

```

Comment:
This will return the kuserid and not the puser id 
Need to use kCurrentContext::$uid